game/external/RUNME.py

```diff
@@ -0,0 +1,18 @@
+import os
+import shutil
+import json
+
+manifest = open("manifest.txt", "r")
+files = manifest.read()
+manifest.close()
+files = json.loads(files)
+files.pop("example.file", "return")
+
+for file in files:
+    if not os.path.exists(file):
+        raise Exception("file '{0}' does not exist".format(file))
+    if not os.path.exists(files[file]):
```

Comment:
Can we allow this to be a list as well? It's possible we might want to copy things to multiple locations as well

(Perhaps build a class to handle the options and we can `execute` the appropriate operations from the class itself?